doc/about.rst

```diff
@@ -0,0 +1,47 @@
+.. _about:
+
+======================
+About ``mybinder.org``
+======================
+
+This page explains some of the teams and organizations behind mybinder.org.
```

Comment:
I like that this starts with "teams and orgs behind mybinder.org" so I would swap the order of the sections below and put the "tech" one at the end. Right now it is a bit weird to talk about humans and then lead with tech :)

Reply:
Agree with @betatim